spec/models/manageiq/providers/lenovo/physical_infra_manager_spec.rb

```diff
@@ -4,6 +4,7 @@
 describe ManageIQ::Providers::Lenovo::PhysicalInfraManager do
   before :all do
     @auth = { :user => 'admin', :pass => 'smartvm', :host => 'localhost', :port => '3000' }
+    FactoryGirl.create(:miq_server, :version=>'master')
   end
 
```

Comment:
Isn't interesting add a test to check the `:user_agent_label`, I mean, is it possible?